scripts/greetings.coffee

```diff
@@ -1,9 +1,23 @@
+# Description:
+#   Greets users when they enter a room. Says goodbye when they leave.
+#
+# Commands:
+#   Enter or exit a room and hubot will say something.
+#
+
 module.exports = (robot) ->
-  enterReplies = require('../stubs/enterReplies')
-  leaveReplies = require('../stubs/leaveReplies')  
+  enterReplies = 
+    ['Hi', 'Target Acquired', 'Hello friend.', 'Gotcha', 'I see you', 'Welcome']
+  leaveReplies = 
+    ['Are you still there?', 'Target lost', 'Searching', 'Hasta Luego']
+  
+  robot.brain.set "enterReplies", enterReplies
+  robot.brain.set "leaveReplies", leaveReplies
```

Comment:
If the messages are edited this seems like it would overwrite them would it be better to get the current values and merge them? (possibly just use https://lodash.com/docs/4.17.4#merge)